src/poliastro/tests/tests_twobody/test_perturbations.py

```diff
@@ -168,3 +169,57 @@ def test_3rd_body_Curtis(test_params):
                               (argp_f * u.rad).to(u.deg) - test_params['orbit'][4]],
                              [test_params['raan'], test_params['inc'], test_params['argp']],
                              rtol=1e-1)
+
+
+solar_pressure_checks = [{'t_days': 200, 'deltas_expected': [3e-3, -8e-3, -0.035, -80.0]},
```

Comment:
Can't we use `pytest.mark.parametrize` instead?

Reply:
No, we can't, because we first calculate trajectory and then check it on some special time values. If we calculate trajectory multiple times, testing would take tooooo long

Reply:
OK, I understand. I think this could be achieved more cleanly using `xunit` setup functions https://docs.pytest.org/en/latest/xunit_setup.html or fixtures https://docs.pytest.org/en/latest/fixture.html#scope-sharing-a-fixture-instance-across-tests-in-a-class-module-or-session but let's not block the pull request because of this.